frontend/src/components/filter/FilterStateSelect.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import FilterSelect from './FilterSelect';
+import { getStateCodes } from '../../fetchers/users';
+
+export default function FilterStateSelect({
+  onApply,
+  inputId,
+  query,
+}) {
+  const [stateCodes, setStateCodes] = useState([]);
+
+  // fetch state codes for user
+  useEffect(() => {
+    async function fetchStateCodes() {
+      try {
+        const codes = await getStateCodes();
+        setStateCodes(codes);
+      } catch (error) {
+        // fail silently
```

Comment:
We probably want to show some type of error if we can't retrieve the states. After think some more about this I wonder if the better solution is to just show all states for regions 11/12 and have a set list for each other region.